README.md

```diff
@@ -1,47 +1,42 @@
 # qbx_garages
 
-**Public Garages**
-* Park owned cars in public garages.
-* You can only parks vehicles that you own in public garages. 
+Provides an API to store and retrieve vehicles from various locations around the map. Also includes an optional built-in UI
 
-![image](https://user-images.githubusercontent.com/82112471/149678987-02ec660f-76c9-4414-af7b-bac284ed58b7.png)
+## Features
 
-![image](https://user-images.githubusercontent.com/82112471/149678977-2a574ee9-8ecc-494f-a845-e17281a74594.png)
+### Built-in UI
+Optionally disabled via config. Created using ox_lib
 
+![Garage Menu](example-menu.png)
```

Comment:
These should be in the .github folder so we don't have to remove them from release versions.

Reply:
done